variables.tf

```
@@ -1,3 +1,9 @@
+variable "region" {
+  description = "The AWS region to create resources in."
+  type        = string
+  default     = "us-east-1"
```

will this be better?

```suggestion
  default     = "ap-southeast-1"
```